testsupport/tiers/checks.go

```diff
@@ -30,6 +30,7 @@ const (
 	appstudioEnv    = "appstudio-env"
 	base            = "base"
 	base1ns         = "base1ns"
+	gobase1ns       = "base1ns-gotemplate"
```

Comment:
minor - but just for consistency we could name this `base1nsGOTemplate` . I see all the other variables follow the tier name.